README.md

```diff
@@ -1,16 +1,38 @@
-# lk2nd
-lk2nd is a bootloader for Qualcomm MSM devices (at the moment only MSM8916 and MSM8939),
-based on the [CodeAurora Little Kernel fork](https://source.codeaurora.org/quic/la/kernel/lk/).
-It provides an Android Fastboot interface on devices where the stock bootloader
-does not provide Fastboot (e.g. Samsung).
-
-On MSM8916 it is also used for some quirks for mainline devices, e.g. to set
-a proper WiFi/BT MAC address in the device tree.
-
-lk2nd does not replace the stock bootloader. It is packaged into an Android
-boot image and then loaded by the stock bootloader as a "secondary" bootloader.
-The real Android boot image is placed into the boot partition with 512 KiB offset,
-and then loaded by lk2nd.
+# lk2nd (and lk1st)
+lk2nd and lk1st are custom bootloaders for many different devices based on
+Qualcomm system-on-chips (SoCs), including smartphones, tablets, smartwatches
+and a few single board computers (SBCs). They simplify booting standard
+operating systems by providing:
+
+  - **A unified boot and flashing interface** (Android Fastboot), independent of
+    device-specific modifications made to the boot chain (e.g. the proprietary
+    flashing interface on Samsung devices)
+  - **Automatic device and hardware detection** (e.g. display panels, batteries, ...)
+  - **Additional bootloader features** (e.g. file system boot, booting from SD card)
+  - **Additional firmware features** (e.g. booting secondary CPU cores if the firmware
+    lacks [PSCI] support)
+  - **Workarounds for various device-specific issues** (e.g. USB on some Samsung
+    devices, modem on Motorola smartphones, ...)
+  - **Various debugging commands** that allow better understanding of the hardware
```

Comment:
```suggestion
  - **Various debugging commands** that allow better understanding of the hardware
      for research and development purposes
```

clarification seems useful there as this is a rather niche feature that few people actually will rely on.